components/clarinet-deployments/src/lib.rs

```diff
@@ -322,7 +322,11 @@ pub async fn generate_default_deployment(
                     ))
                 }
             };
-            queue.push_front(contract_id)
+            queue.push_front((
+                contract_id,
+                StacksEpochId::Epoch21,
+                ClarityVersion::Clarity2,
```

Comment:
Do you mean to add new parameters in the manifest for the subnet's epoch and Clarity version, or use some other existing value?

Reply:
Yeah sorry my comment doesn't actually makes much sense.
I think that hard coded epodchIDs and ClarityVersions mean that we'll have to refactor it at some point (to handle futures versions / use cases). So I was wondering if it could be handled dynamically here.
But maybe it doesn't make sense

Reply:
I see what you mean. Maybe it could be as simple as just using `::latest()`? Presumably, in testing, we're always going to want to be testing the subnet contract on the latest.

Reply:
> Presumably, in testing, we're always going to want to be testing the subnet contract on the latest.

Even when Clarity 3 will be in thing?

Reply:
I was thinking that if you are using an existing subnet contract, that's live, then you wouldn't have this issue, and this case would only be for new contracts, but I guess that's not right. It will always deploy this way. 

So maybe it should instead be in the docker image, with the contract. I could just also have the .json file that this PR adds to the cache in the image.

Reply:
Well maybe it's too early too know what the future of subnets will be.
Feel free to mark as resolved if you think it's not necessary to address it now

Reply:
Ok, yeah, let's come back to this later.